src/app/bridge-btc/page.tsx

```diff
@@ -346,39 +315,32 @@ const BridgeBtc = () => {
 	const [contract, setContract] = useState<Contract>(new Contract());
 	const [tokens, setTokens] = useState<Token[]>([]);
 	useEffect(() => {
-		lg("useEffect() ... setTokens");
 		const run = async () => {
 			const tokens = await contract.allTokensWithInfo();
-			lg("tokens:", tokens);
+			//console.log('tokens:', tokens)
 			setTokens(tokens);
 			const tokenObj = tokens.find((token) => token.symbol === tokenSymbol);
 			if (!tokenObj) {
 				console.error("No_token_found");
 				return;
 			}
 			setSelectedToken(tokenObj);
-			lg("tokenObj:", tokenObj);
+			console.log("tokenObj:", tokenObj);
```

Comment:
@AuroraLantean I don't see much value in having console logs in production at all.